modules/uniref90_rhea_filter/main.nf

```diff
@@ -0,0 +1,14 @@
+process UNIREF90_RHEA_FILTER {
+    label 'process_single'
+    container 'quay.io/biocontainers/biopython:1.78'
+
+    input:
+    path uniref90_fasta
+    path uniprot_rhea_mapping
+
+    output:
+    path 'uniref90_with_rhea.fasta', emit: filtered_fasta
+
+    script:
+    "uniref90_rhea_filter.py ${uniref90_fasta} ${uniprot_rhea_mapping} uniref90_with_rhea.fasta"
```

Comment:
better use """ code """ instead of " line ", for nf-style consistency

Reply:
Done!